spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -74,7 +74,12 @@ An alternative is selected for the bean archive if either:
 ----
 
 Each child `<class>` element must specify the name of a bean class of an alternative bean.
```

Comment:
I removed the first sentence and merged it with the 2nd. Should make things clearer. The first sentence was redundant with the paragraph above anyway.